view/admin.phtml

```diff
@@ -30,9 +30,9 @@
  */
 ?><div class="wrap">
     <?php if (! extension_loaded('ldap')) : ?>
-    <div class="error"><strong>Caveat:</strong> The LDAP-extension is not loaded!
```

Comment:
Why did you remove the strong marking here?

Reply:
This was more of a personal opinion, the alert is already inside a <div class="error"/> element so it seemed unnecessary to add extra emphasis on any text